docs/getting_started/setup_folder_rights.md

```diff
@@ -0,0 +1,78 @@
+# Setup Folder Rights
+
+Like most things, [Symfony documentation](http://symfony.com/doc/3.4/setup/file_permissions.html) applies here, in this case meaning `var`, `web/var` need to be writable by cli and web server user.
+Furthermore, future files and directories created by these two users will need to inherit those access rights. 
+
+!!! caution
+    For security reasons, in production there is no need for web server to have access to write to other directories.
+
+!!! tip "Development Setup"
+    For development setup you may change your web server config to use same user as owner of folder, what follows below are mainly for production setup, and like Symfony we first and foremost recommend using an approach using ACL.
+
+## A. **Using ACL on a *Linux/BSD* system that supports chmod +a**
+
+Some systems, primarily Mac OS X, supports setting ACL using a `+a` flag on `chmod`. Example uses a command to
+try to determine your web server user and set it as ``HTTPDUSER``, alternatively change to your actual web server
+user if non standard:
+
+```bash
+$ rm -rf var/cache/* var/logs/* var/sessions/*
+$ HTTPDUSER=$(ps axo user,comm | grep -E '[a]pache|[h]ttpd|[_]www|[w]ww-data|[n]ginx' | grep -v root | head -1 | cut -d\  -f1)
+$ sudo chmod +a "$HTTPDUSER allow delete,write,append,file_inherit,directory_inherit" var web/var
+$ sudo chmod +a "$(whoami) allow delete,write,append,file_inherit,directory_inherit" var web/var
+```
+
+## B. **Using ACL on a *Linux/BSD* system that does not support chmod +a**
+
+Some systems don't support chmod +a, but do support another utility called setfacl. You may need to
```

Comment:
`Some systems (for example Ubuntu)`
Then remove `*(as is the case with Ubuntu)*` below.